framework/src/seclink/seclink.c

```diff
@@ -166,13 +180,21 @@ int sl_set_key(sl_ctx hnd, hal_key_type mode, uint32_t key_idx, hal_data *key, h
 
 	SL_CHECK_VALID(hnd);
 
+	struct timespec start, end;
+	clock_gettime(CLOCK_REALTIME, &start);
+
 	struct _seclink_s_ *sl = (struct _seclink_s_ *)hnd;
 	struct seclink_key_info info = {mode, key_idx, key, prikey};
 	struct seclink_req req = {.req_type.key = &info, 0};
 
 	SL_CALL(sl, SECLINKIOC_SETKEY, req);
 	*hres = req.res;
 
+	clock_gettime(CLOCK_REALTIME, &end);
+	uint32_t elapsed = _calc_elasped(&start, &end);
+	printf("[pkbuild] %s mode %d idx %d res %d elpased %u %u\n",
+		   __FUNCTION__, mode, key_idx, *hres, elapsed, elapsed/1000000);
```

Comment:
Please add space before and after `/`.

Reply:
And I think it would be better to add unit to logs like below.
```
"__FUNCTION__ mode # idx # res # elpased ####(ns) ##(s)"
```